src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -333,4 +334,36 @@ std::map<uint64_t, std::string> FindFunctionsThatUprobesCannotInstrumentWithMess
   return function_ids_to_error_messages;
 }
 
+absl::flat_hash_map<pid_t, pid_t> RetrieveInitialTidToRootNamespaceTidMapping(pid_t target_pid) {
```

Comment:
I would change `target_pid` to `pid_in_root_namespace`.

Reply:
done